src/interfaces.ts

```diff
@@ -0,0 +1,55 @@
+export interface LEVLEICredential {
+    i: string;
+    dt: string;
+    LEI: string;
+}
+export interface LE_VLEI_EDGES {
+    d: string,
+    qvi: QVI_VLEI_EDGE
+}
+export interface QVI_VLEI_EDGE {
+    n: string,//issuer of the credential
+    s: "EBfdlu8R27Fbx-ehrqwImnK-8Cm79sqbAQ4MmvEAYqao", //subject of the credential
```

Comment:
A better comment would be `Legal Entity vLEI Credential Schema` or preferably make each known schema SAID a constant in the library, any change in vLEI schemas would require a versioning change here.